docs/content/Schema/pre-aggregations.md

```diff
@@ -285,6 +285,14 @@ Original SQL pre-aggregation is the simplest type of pre-aggregation. As the
 name suggests, it persists the results of the SQL query of the cube in which it
```

Comment:
Let's rewrite this section. We need to say that 
1. original SQL pre-aggregations should be used only with internal: true 
2. they do not provide a lot of performance improvements usually
3. they should be used only for `useOriginalSqlPreAggregations` 
4. or for use cases where it is not possible to use rollups